bmwqemu.pm

```diff
@@ -399,6 +400,18 @@ sub random_string {
     return $string;
 }
 
+# return a short string representing the given string by passing it through
+# the MD5 algorithm and taking the first characters
+sub hashed_string {
+    my ($string, $count) = @_;
+    $count //= 5;
+
+    my $hash = md5_base64($string);
+    # plus sign is problematic in regexps
+    $hash =~ s,\+,_,g;
```

Comment:
If you would use `md5_hex` then there is no need for substitute. Is there a reason to use base64?